src/pages/mesh/advanced/index.md

```diff
@@ -614,3 +614,115 @@ aio api-mesh:source:install "AEM Assets API"@0.0.1
 ```bash
 Successfully updated the mesh with the id: MESH_ID
 ```
+
+## aio api-mesh:log-list
+
+The `log-list` command lists the most recent requests for your mesh by rayID. By default, the command shows the 15 most recent requests.
+
+### Usage
+
+```bash
+aio api-mesh:log-list
```

Comment:
Your usage statements should include the flags. The flag names should be enclosed in brackets if they're options. I checked the [aio command reference}(https://github.com/adobe/aio-cli), and that's what they're doing there. Note that you might need to update all the usage statements in the file.

Reply:
@keharper good catch! I've standardized these based on the `aio` formatting.